testdata/baselines/reference/submodule/compiler/declarationEmitLateBoundAssignments.js.diff

```diff
@@ -1,10 +1,7 @@
 --- old.declarationEmitLateBoundAssignments.js
 +++ new.declarationEmitLateBoundAssignments.js
-@@= skipped -35, +35 lines =@@
-
- //// [declarationEmitLateBoundAssignments.d.ts]
- export declare function foo(): void;
--export declare namespace foo {
--    var bar: number;
--    var strMemName: string;
--}
\ No newline at end of file
+@@= skipped -39, +39 lines =@@
+     var bar: number;
+     var strMemName: string;
+ }
++export {};
```

Comment:
Hm. Wonder why these are appearing so frequently in this change.

Reply:
@jakebailey Looks like these empty exports are related to this typo

https://github.com/microsoft/typescript-go/blob/2b82831a05b6b99da279d12fecbcbc460574a85b/internal/transformers/declarations/transform.go#L268-L270

```go
ast.IsExternalModuleIndicator(elem)
```